.github/workflows/deploy.yml

```diff
@@ -0,0 +1,52 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - master
+      - source
+  pull_request:
+    branches:
+      - master
+      - source
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Setup Ruby
+      uses: actions/setup-ruby@v1
+    - name: Enable bundler cache
+      uses: actions/cache@v2
+      with:
+        path: vendor/bundle
+        key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-gems-
+    - name: Install deps
+      run: |
+        gem install bundler
+        bundle config path vendor/bundle
+        bundle install --jobs 4 --retry 3
+    - name: Setup deploy options
+      id: setup
+      run: |
+        git config --global user.name "GitHub Action"
+        git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
Where is this email coming from? Does it correspond to a specific GitHub action?

Reply:
You can use any email here, `<some_number>+github-actions[bot]@users.noreply.github.com` is the Github Actions default email. See https://github.community/t/github-actions-bot-email-address/17204 for discussion.